android/java/org/chromium/chrome/browser/BraveRewardsPanelPopup.java

```diff
@@ -1626,23 +1626,32 @@ private void SetAddFundsBtnClickHandler(@WalletStatus final int status) {
             public void onClick(View v) {
                 dismiss();
                 mBraveActivity.openNewOrSelectExistingTab (mExternal_wallet.mAdd_url);
-        }}));
+            }
+        }));
     }
 
     private void SetVerifyWalletBtnClickHandler(@WalletStatus final int status) {
         Button btnVerifyWallet = (Button)root.findViewById(R.id.btn_verify_wallet);
+        BraveRewardsBalance balance_obj = mBraveRewardsNativeWorker.GetWalletBalance();
+        if (balance_obj != null) {
+            walletBalance = balance_obj.mTotal;
+        }
+
         btnVerifyWallet.setOnClickListener((new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                switch(status) {
+                if (walletBalance < 25) {
```

Comment:
let's declare it on top of the class, in case we want to change it after. It's harder to find it, when it's burred inside sources

Reply:
@SergeyZhukovsky good point. updating it.